priv/templates/nimble_template/lib/otp_app_web/controllers/api/fallback_controller.ex.eex

```diff
@@ -0,0 +1,13 @@
+defmodule <%= web_module %>.Api.FallbackController do
+  use Phoenix.Controller
+
+  alias Ecto.Changeset
+  alias <%= web_module %>.Api.ErrorView
+
+  def call(conn, {:error, :invalid_params, %Changeset{valid?: false} = changeset}) do
```

Comment:
Just a question, when this will be called?

Reply:
When the params is invalid on the `controller params` level @junan you can check it in the `params_validator.ex`